Code/RDGeneral/LocaleSwitcher.cpp

```diff
@@ -97,6 +97,7 @@ class LocaleSwitcherImpl {
     if (!recurseLocale(CurrentState)) {
 #ifdef RDK_THREADSAFE_SSS
       _configthreadlocale(_ENABLE_PER_THREAD_LOCALE);
+      old_locale = ::setlocale(LC_ALL, nullptr);
```

Comment:
this is a stupid question (for which I apologize), but why isn't this `std::setlocale`?

Reply:
I kept as it was. I assume the idea is to call the Windows native `setlocale` rather than `std::setlocale`. I haven't investigated if it makes a difference - I just kept the previous version to keep changes to a minimum.

Reply:
BTW, I see that I didn't branch off from master, but from the previous branch. I'll close this one and open a new one.